src/database.ts

```diff
@@ -134,7 +134,7 @@ class Database {
    * @param {function} [callback]
    * @return {Promise}
```

Comment:
```suggestion
   * @return {BluebirdPromise}
```

Reply:
IMHO we can remove those JSDoc altogether and only uses TypeScript types.